Autopilot/Path Manager/MainBatterySensor.c

```diff
@@ -6,40 +6,34 @@
  */
 #include "main.h"
 #include "../Common/Common.h"
-#include "voltageSensor.h"
+#include "MainBatterySensor.h"
```

Comment:
Ok, i'm fine with this naming then. Maybe consider naming ExtBattery to MotorBattery to make it really clear when you get to implementing it.